packages/@vue/cli/package.json

```diff
@@ -48,6 +48,7 @@
     "lodash.clonedeep": "^4.5.0",
     "lru-cache": "^5.1.1",
     "minimist": "^1.2.0",
+    "read-pkg": "^5.1.1",
```

Comment:
You don't need this dep. We have some tools that gets the pkg info.

Reply:
I'm not sure which tool I should use. Currently I only found that [@vue/cli-service/lib/Service.js#L49](https://github.com/vuejs/vue-cli/blob/v4.1.1/packages/%40vue/cli-service/lib/Service.js#L49) uses `read-pkg` to load the `package.json`.

Reply:
Yup, so move this resolvepkg function to utils package, and we can use it as common in both places instead of doing it like this.

Reply:
That makes sense.